src/components/modules/blog/code/index.tsx

```diff
@@ -0,0 +1,21 @@
+import { Roboto_Mono } from 'next/font/google';
+
+import { StyledBlogCodeContent, StyledBlogCodeTitleWrapper, StyledBlogCodeWrapper } from './code.styles';
+import type { BlogCodeProps } from './code.types';
+
+import { Text } from '@/components/elements/text';
+
+const dmCodeFont = Roboto_Mono({ subsets: ['latin'], weight: ['400'], variable: '--font-code' });
+
+export const BlogCode = ({ title, content }: BlogCodeProps) => {
```

Comment:
@RBoraurito, Can we name this more generic, something like `BlockCode`. Probably this component will be used also in documentation & fag pages not only in blog content.